src/types/plugin-input.ts

```diff
@@ -13,6 +13,8 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
 
 const ONE_DAY = 24 * 60 * 60 * 1000;
 
+const userRoleSchema = T.Union([T.Literal("Admin"), T.Literal("Member"), T.Literal("Contributor"), T.String()]);
```

Comment:
is this used anywhere? The literals differ from the typebox defaults so if it's used sync them up otherwise remove it